content/en/blog/_posts/2018-05-29-announcing-kustomize.md

```diff
@@ -8,12 +8,11 @@ date: 2018-05-29
 
 [**kustomize**]: https://github.com/kubernetes-sigs/kustomize
 [hello world]: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/helloWorld
-[kustomization]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization
```

Comment:
I deleted [kustomization] because it's outdated link that leads to nowhere, and it's not used in the main content.